docs/stats.md

```diff
@@ -0,0 +1,38 @@
+# Stats
+
+## Overview
+<!-- TODO: move module definitions here? -->
+A set of methods used to calculate statistics over the course of an
+experiment.
+Each such method is a higher-order function returning a `track_stat` method
+that takes in an Experiment object, an action key, and the address of the
```

Comment:
mm you might be looking at outdated code, the `track_stat` method takes in a `state` object (from an `Experiment`'s `run` method), and an `action` object.